apps/frontend/src/components/timeout/TimeOutPrompt.tsx

```diff
@@ -1,30 +1,28 @@
 import {
   Button,
-  Dialog,
   DialogActions,
   DialogContent,
   DialogContentText,
-  DialogTitle,
 } from '@mui/material';
 import React from 'react';
 
+import StyledDialog from 'components/common/StyledDialog';
+
 type IdleTimeoutPromptProps = {
   isIdle: boolean;
   onConfirm: () => void;
 };
 
 function IdleTimeoutPrompt(props: IdleTimeoutPromptProps): JSX.Element {
   return (
-    <Dialog fullWidth data-cy="timeout-dialog" open={props.isIdle}>
-      <DialogTitle
-        sx={(theme) => ({
-          marginTop: theme.spacing(1.5),
-          color: theme.palette.error.main,
-        })}
-      >
-        {'Warning'}
-      </DialogTitle>
-      <DialogContent>
+    <StyledDialog
+      fullWidth
```

Comment:
Would you mind adding the error colouring back to the title?

Reply:
@TCMeldrum It has been sorted out and added back.